themes/digital.gov/layouts/partials/core/usa-banner.html

```diff
@@ -1,6 +1,6 @@
 {{- $uswds_img_path := "/uswds/img" -}}
 <a class="usa-skipnav" href="#main-content">Skip to main content</a>
-<div class="usa-banner gov-banner">
+<section class="usa-banner gov-banner">
```

Comment:
I thought we already updated banner markup?

Reply:
This code shouldn't have been included. It was for another branch, sorry about that!